vyxal/elements.py

```diff
@@ -2379,6 +2379,24 @@ def matrix_determinant(lhs, ctx):
     return sympy.det(sympy.Matrix(lhs))
 
 
+def matrix_exponentiation(lhs, rhs, ctx):
+    """Element Þe
+    (lst, num) -> (a * a) b times
+    (num, lst) -> (b * b) a times
+    """
+
+    ts = vy_type(lhs, rhs, simple=True)
+    if set(ts) != {list, NUMBER_TYPE}:
+        raise TypeError("matrix exponentiation only works on lists and numbers")
```

Comment:
Maybe say instead "Matrix exponentiation requires a matrix and a number"? This implies that two numbers or two matrices would work